go/coinstacks/thorchain/api/affiliateFees.go

```diff
@@ -245,19 +245,19 @@ func (i *AffiliateFeeIndexer) handleBlock(httpClient *cosmos.HTTPClient, block *
 		Block: block,
 	}
 
-	i.processAffiliateFees(b, blockResult.EndBlockEvents, []string{affiliateAddress})
+	i.processAffiliateFees(b, blockResult.FinalizeBlockEvents, []string{affiliateAddress})
 }
 
 func (i *AffiliateFeeIndexer) handleNewBlockHeader(newBlockHeader types.EventDataNewBlockHeader) {
-	b := &thorchain.NewBlockHeader{
-		EventDataNewBlockHeader: newBlockHeader,
-	}
+	//b := &thorchain.NewBlockHeader{
```

Comment:
Block events are no longer on the block header. Logic needs to be updated to either subscribe accordingly (desired to avoid extra request) or do a subsequent request to get block events (fallback strategy)